api/src/main/scala/za/co/absa/loginsvc/rest/config/auth/ActiveDirectoryLDAPConfig.scala

```diff
@@ -27,7 +27,7 @@ import za.co.absa.loginsvc.rest.config.validation.{ConfigValidatable, ConfigVali
  * @param url URL to AD LDAP, ex. "ldaps://some.domain.com:636/"
  * @param searchFilter LDAP filter used when searching for groups, ex. "(samaccountname={1})"
  */
-case class ActiveDirectoryLDAPConfig(domain: String, url: String, searchFilter: String, order: Int)
+case class ActiveDirectoryLDAPConfig(domain: String, url: String, searchFilter: String, order: Int, attributes: Option[Array[String]])
```

Comment:
Is this necessary or is there a way to reasonably load no attributes as an empty array?

```suggestion
case class ActiveDirectoryLDAPConfig(domain: String, url: String, searchFilter: String, order: Int, attributes: Array[String])
```

Reply:
So for PureConfig, if you want the entire tag to be optional then it needs to be an option. Otherwise, we can get rid of the Option but then the tag is mandatory in the config and an empty array has to be supplied if you're not using it. So I guess it's a toss-up between having the field be mandatory in the config or not?